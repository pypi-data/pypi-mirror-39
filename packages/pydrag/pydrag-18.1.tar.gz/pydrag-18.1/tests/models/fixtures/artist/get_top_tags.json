{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&artist=Guns+N%27+Roses&autocorrect=1&format=json&mbid=eeb1195b-f213-4ce1-b28c-8565211f8e43&method=artist.getTopTags"
            },
            "response": {
                "body": {
                    "string": "{\"toptags\":{\"tag\":[{\"count\":100,\"name\":\"hard rock\",\"url\":\"https://www.last.fm/tag/hard+rock\"},{\"count\":70,\"name\":\"rock\",\"url\":\"https://www.last.fm/tag/rock\"},{\"count\":55,\"name\":\"classic rock\",\"url\":\"https://www.last.fm/tag/classic+rock\"},{\"count\":38,\"name\":\"80s\",\"url\":\"https://www.last.fm/tag/80s\"},{\"count\":24,\"name\":\"metal\",\"url\":\"https://www.last.fm/tag/metal\"},{\"count\":11,\"name\":\"heavy metal\",\"url\":\"https://www.last.fm/tag/heavy+metal\"},{\"count\":11,\"name\":\"seen live\",\"url\":\"https://www.last.fm/tag/seen+live\"},{\"count\":5,\"name\":\"Guns N Roses\",\"url\":\"https://www.last.fm/tag/Guns+N+Roses\"},{\"count\":5,\"name\":\"american\",\"url\":\"https://www.last.fm/tag/american\"},{\"count\":4,\"name\":\"hair metal\",\"url\":\"https://www.last.fm/tag/hair+metal\"},{\"count\":4,\"name\":\"90s\",\"url\":\"https://www.last.fm/tag/90s\"},{\"count\":4,\"name\":\"glam rock\",\"url\":\"https://www.last.fm/tag/glam+rock\"},{\"count\":3,\"name\":\"alternative\",\"url\":\"https://www.last.fm/tag/alternative\"},{\"count\":3,\"name\":\"Glam Metal\",\"url\":\"https://www.last.fm/tag/Glam+Metal\"},{\"count\":2,\"name\":\"Guns N' Roses\",\"url\":\"https://www.last.fm/tag/Guns+N%27+Roses\"},{\"count\":2,\"name\":\"Slash\",\"url\":\"https://www.last.fm/tag/Slash\"},{\"count\":2,\"name\":\"rock n roll\",\"url\":\"https://www.last.fm/tag/rock+n+roll\"},{\"count\":2,\"name\":\"USA\",\"url\":\"https://www.last.fm/tag/USA\"},{\"count\":2,\"name\":\"sleaze rock\",\"url\":\"https://www.last.fm/tag/sleaze+rock\"},{\"count\":2,\"name\":\"alternative rock\",\"url\":\"https://www.last.fm/tag/alternative+rock\"},{\"count\":2,\"name\":\"guitar\",\"url\":\"https://www.last.fm/tag/guitar\"},{\"count\":2,\"name\":\"Axl Rose\",\"url\":\"https://www.last.fm/tag/Axl+Rose\"},{\"count\":2,\"name\":\"male vocalists\",\"url\":\"https://www.last.fm/tag/male+vocalists\"},{\"count\":2,\"name\":\"punk\",\"url\":\"https://www.last.fm/tag/punk\"},{\"count\":1,\"name\":\"blues rock\",\"url\":\"https://www.last.fm/tag/blues+rock\"}],\"@attr\":{\"artist\":\"Guns N' Roses\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "1875"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 15:02:26 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
