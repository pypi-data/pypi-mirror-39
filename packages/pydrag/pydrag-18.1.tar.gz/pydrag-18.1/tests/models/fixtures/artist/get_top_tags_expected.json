{
    "artist": "Guns N' Roses",
    "data": [
        {
            "count": 100,
            "name": "hard rock",
            "url": "https://www.last.fm/tag/hard+rock"
        },
        {
            "count": 70,
            "name": "rock",
            "url": "https://www.last.fm/tag/rock"
        },
        {
            "count": 55,
            "name": "classic rock",
            "url": "https://www.last.fm/tag/classic+rock"
        },
        {
            "count": 38,
            "name": "80s",
            "url": "https://www.last.fm/tag/80s"
        },
        {
            "count": 24,
            "name": "metal",
            "url": "https://www.last.fm/tag/metal"
        },
        {
            "count": 11,
            "name": "heavy metal",
            "url": "https://www.last.fm/tag/heavy+metal"
        },
        {
            "count": 11,
            "name": "seen live",
            "url": "https://www.last.fm/tag/seen+live"
        },
        {
            "count": 5,
            "name": "Guns N Roses",
            "url": "https://www.last.fm/tag/Guns+N+Roses"
        },
        {
            "count": 5,
            "name": "american",
            "url": "https://www.last.fm/tag/american"
        },
        {
            "count": 4,
            "name": "hair metal",
            "url": "https://www.last.fm/tag/hair+metal"
        },
        {
            "count": 4,
            "name": "90s",
            "url": "https://www.last.fm/tag/90s"
        },
        {
            "count": 4,
            "name": "glam rock",
            "url": "https://www.last.fm/tag/glam+rock"
        },
        {
            "count": 3,
            "name": "alternative",
            "url": "https://www.last.fm/tag/alternative"
        },
        {
            "count": 3,
            "name": "Glam Metal",
            "url": "https://www.last.fm/tag/Glam+Metal"
        },
        {
            "count": 2,
            "name": "Guns N' Roses",
            "url": "https://www.last.fm/tag/Guns+N%27+Roses"
        },
        {
            "count": 2,
            "name": "Slash",
            "url": "https://www.last.fm/tag/Slash"
        },
        {
            "count": 2,
            "name": "rock n roll",
            "url": "https://www.last.fm/tag/rock+n+roll"
        },
        {
            "count": 2,
            "name": "USA",
            "url": "https://www.last.fm/tag/USA"
        },
        {
            "count": 2,
            "name": "sleaze rock",
            "url": "https://www.last.fm/tag/sleaze+rock"
        },
        {
            "count": 2,
            "name": "alternative rock",
            "url": "https://www.last.fm/tag/alternative+rock"
        },
        {
            "count": 2,
            "name": "guitar",
            "url": "https://www.last.fm/tag/guitar"
        },
        {
            "count": 2,
            "name": "Axl Rose",
            "url": "https://www.last.fm/tag/Axl+Rose"
        },
        {
            "count": 2,
            "name": "male vocalists",
            "url": "https://www.last.fm/tag/male+vocalists"
        },
        {
            "count": 2,
            "name": "punk",
            "url": "https://www.last.fm/tag/punk"
        },
        {
            "count": 1,
            "name": "blues rock",
            "url": "https://www.last.fm/tag/blues+rock"
        }
    ]
}
